### PR TITLE
support tpc-h

### PR DESCRIPTION
**Pull request by wzru**

### CODE REVIEW

File: flows/tpch.load-bench.tiflow [hunk 1]
```diff
@@ -0,0 +1,2 @@
+help = reset cluster, load TPCC and run
```
Comment (innerr) [inline flows/tpch.load-bench.tiflow:1]: TPCC => TPCH

File: flows/tpch.test.config.tiflow [hunk 2]
```diff
@@ -0,0 +1,4 @@
+help = setup all configs for quick test @config
+abbr = tpch.test.conf|cfg
+
+flow = tidb.yaml.template test : tidb.set-confirm no : tpch.config.min
```
Comment (innerr) [inline flows/tpch.test.config.tiflow:4]: Don't want `test` have tiflash, it slows things down

File: tpch/gen-backup-tag.bash [hunk 3]
```diff
@@ -0,0 +1,8 @@
+set -euo pipefail
```
Comment (innerr) [inline tpch/gen-backup-tag.bash:1]: File name not right, should be gen-tag.bash

File: tpch/gen-backup-tag.bash.ticat [hunk 4]
```diff
@@ -0,0 +1,6 @@
+help = generate backup tag name by data scale
```
Comment (innerr) [inline tpch/gen-backup-tag.bash.ticat:1]: File name not right, should be gen-tag.bash.ticat
Help not right, this should only generate `bench.workload.tag`

File: tpch/run.bash.ticat [hunk 5]
```diff
@@ -0,0 +1,31 @@
+help = run TPCH benchmark
+abbr = r|R
```
Comment (innerr) [inline tpch/run.bash.ticat:2]: Too short, remove abbr

Comment (wzru) [reply tpch/run.bash.ticat:2]: refer to 
https://github.com/innerr/bench.tidb.ticat/blob/15d4457ffcd90b71eadbb546184a8d845ee67d93/tpcc/run.bash.ticat#L2

Comment (wzru) [reply tpch/run.bash.ticat:2]: refer to https://github.com/innerr/bench.tidb.ticat/blob/15d4457ffcd90b71eadbb546184a8d845ee67d93/tpcc/run.bash.ticat#L2

Comment (wzru) [reply tpch/run.bash.ticat:2]: refer to https://github.com/innerr/bench.tidb.ticat/blob/15d4457ffcd90b71eadbb546184a8d845ee67d93/tpcc/run.bash.ticat#L2

Comment (wzru) [reply tpch/run.bash.ticat:2]: refer to 
https://github.com/innerr/bench.tidb.ticat/blob/15d4457ffcd90b71eadbb546184a8d845ee67d93/tpcc/run.bash.ticat#L2

File: tpch/load.bash.ticat [hunk 6]
```diff
@@ -0,0 +1,26 @@
+help = prepare TPCC data
```
Comment (innerr) [inline tpch/load.bash.ticat:1]: TPCC

File: tpch/run.bash.ticat (hunk 5 above)
Comment (innerr) [reply tpch/run.bash.ticat:2]: Remove them both

Comment (innerr) [reply tpch/run.bash.ticat]: Second thought, might be OK, the path contains the workload name, small chance to conflict

**Review by innerr — APPROVED** [review]

**Pull request merged**